site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -165,12 +165,37 @@ We need a firewall to allow UDP traffic to nodes tagged as `game-server` via por
 next section.
 
 ```bash
-gcloud compute firewall-rules create game-server-firewall \
+gcloud compute firewall-rules create gke-agones-game-server-firewall \
   --allow udp:7000-8000 \
   --target-tags game-server \
   --description "Firewall to allow game server udp traffic"
 ```
 
+### Create an Autopilot mode cluster for Agones
```

Comment:
You moved this up above "Creating a cluster", so now it appears as a subheading of "Creating the firewall". I think this block instead belongs at old line 180, swapping the two.
![image](https://github.com/googleforgames/agones/assets/4942464/cf89a0df-e8a4-4b80-9c5a-c32552eb9e33)

Reply:
Ohh my bad, i just swapped them again